### DISCUSSION TITLE
Connect tasmota to android device from app and get device info while sending wifi info

### DISCUSSION THREAD

**Discussion by b1rigu** [arendst/Tasmota#14585] [Q&A] [answered]
I have been digging around pretty much all day and can't find anything that really hits the spot. Basically what I'm trying to achieve is an app that searches for locally available tasmota device, get the device info, saves the device info, and sends wifi ssid and password to the device. I am using android studio with java and devices with tasmota. Is there any configuration I can do in tasmota firmware to achieve this? any help is appreciated. thanks.

**Comment by Jason2866** [comment, marked as answer]
When using vanilla Tasmota builds you will see all devices at IP address 192.168.4.1 with the AP name `Tasmota-<xxxx>`

**Reply by b1rigu** [reply]
Yeah but I wanna make that hidden so I could only access it from phone and control it from there. Basically an all around fully automated pairing system. Only user input is the ssid and pass of currently connected wifi. Any ideas on that?

**Reply by Jason2866** [reply]
No, not with standard Tasmota. If you want such a function you have to fork and implement.

**Reply by b1rigu** [reply]
Oh okay. Can you let me know if you know how I can implement and program to make it like that. Maybe some type of wifi protocol that I can use. Any info will be helpful

**Reply by barbudor** [reply]
The only way to configure a freshly flashed Tasmota is either:
- Serial
- Let it open its AP `tasmota_xxxxx`, connects to it and use the web interface or web API

So you have to scan for all WIfi `tasmota_xxxx`, connect to it and send the configuration command `http://192.168.4.1/cm?cmnd=backlog%20ssid1%20XXXXXX%3Bpassword1%20YYYYYY`
